src/app/marketplace/marketplace-task-manager/marketplace-task-manager.component.spec.ts

```diff
@@ -54,44 +62,19 @@ describe('MarketplaceTaskManagerComponent', () => {
     });
   });
 
-  describe('#openDialog()', () => {
-    it('should set opened to true', () => {
-      component.opened = false;
-      component.openDialog();
-      expect(component.opened).toBe(true);
-    });
-  });
-
   describe('#removeTask()', () => {
     it('should call taskManagerService.removeTask', () => {
-      const task = { progress: 100 } as Task;
-      component.setTaskManagerService(taskManagerService);
-
+      const task = {} as Task;
       component.removeTask(task);
-
       expect(taskManagerService.removeTask.calls.count()).toBe(1);
     });
   });
 
-  describe('#setTaskMangerService()', () => {
-    it('should set taskManagerService', () => {
-      const unsubscribeTasks = jasmine.createSpy();
-      component[ 'unsubscribeTasks' ] = unsubscribeTasks;
-      component.setTaskManagerService(taskManagerService);
-      expect(component.taskManagerService).toBe(taskManagerService);
-      expect(component.tasks).toEqual(<any> tasks);
-      expect(unsubscribeTasks.calls.count()).toBe(1);
-    });
-  });
-
-  describe('#unsubscribeTasks()', () => {
-    it('should call unsubscribe when there is subscription', () => {
-      const unsubscribe = jasmine.createSpy();
-      component[ 'tasksSubscription' ] = <any> {
-        unsubscribe
-      };
-      component[ 'unsubscribeTasks' ]();
-      expect(unsubscribe.calls.count()).toBe(1);
+  describe('#ngOnDestroy()', () => {
+    it('should unsubscribe all subscriptions', () => {
```

Comment:
Test without any assertions? If it only checks if component exists and can be initialised it shouldn't have name 'should unsubscribe all subscriptions'.